charts/filebeat/values.yaml

```diff
@@ -89,6 +89,22 @@ filebeatConfig:
 
       processors:
         - add_cloud_metadata: ~
+        - rename:
+            fields:
+            - from: "agent"
+              to: "winlogbeat_agent"
+            ignore_missing: true
```

Comment:
If this part is for linux containers, why are we renaming it to **winlogbeat_agent**?

Reply:
this should be filebeat_agent as agent is reserved as a string